multi-node/config/wazuh_cluster/wazuh_manager.conf

```diff
@@ -114,7 +114,6 @@
     <!-- Debian OS vulnerabilities -->
```

Comment:
`jammy` is missing for `canonical` (line 110).

Reply:
done